patch-package
new file mode 100644
Binary files /dev/null and b/node_modules/react-native-interactable/lib/.DS_Store differ
new file mode 100644
Binary files /dev/null and b/node_modules/react-native-interactable/lib/android/.DS_Store differ
deleted file mode 100644
--- a/node_modules/react-native-interactable/lib/android/src/androidTest/java/com/wix/interactable/ExampleInstrumentedTest.java
+++ /dev/null
@@ -1,26 +0,0 @@
-package com.wix.interactable;
-
-import android.content.Context;
-import android.support.test.InstrumentationRegistry;
-import android.support.test.runner.AndroidJUnit4;
-
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import static org.junit.Assert.*;
-
-/**
- * Instrumentation test, which will execute on an Android device.
- *
- * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
- */
-@RunWith(AndroidJUnit4.class)
-public class ExampleInstrumentedTest {
-    @Test
-    public void useAppContext() throws Exception {
-        // Context of the app under test.
-        Context appContext = InstrumentationRegistry.getTargetContext();
-
-        assertEquals("com.wix.interactable", appContext.getPackageName());
-    }
-}
